File: TOMIMSV4/Entity/Mantenimientos/Propietario/Propietario/clsBePropietarios.vb
<Serializable>
Public Class clsBePropietarios
    Implements ICloneable
    Implements IDisposable

    Public Property IdPropietario() As Integer = 0
    Public Property IdEmpresa() As Integer = 0
    Public Property IdTipoActualizacionCosto() As Integer = 0
    Public Property Contacto() As String = ""
    Public Property Nombre_comercial() As String = ""
    Public Property Imagen() As Byte() = Nothing
    Public Property Telefono() As String = ""
    Public Property NIT As String = ""
    Public Property Direccion() As String = ""
    Public Property Activo() As Boolean = False
    Public Property User_agr() As String = ""
    Public Property Fec_agr() As Date = Date.Now
    Public Property User_mod() As String = ""
    Public Property Fec_mod() As Date = Date.Now
    Public Property Email() As String = ""
    Public Property Actualiza_costo_oc() As Boolean = False
    Public Property Color() As Integer = 0
    Public Property Codigo As String = ""
    Public Property Sistema As Boolean = False
    Public Property Es_Consolidador As Boolean = False
    Public Property ControlUx As Boolean = False

    Sub New()
    End Sub

    Sub New(ByRef IdPropietario As Integer, ByVal IdEmpresa As Integer, ByVal IdTipoActualizacionCosto As Integer, ByVal contacto As String, ByVal nombre_comercial As String, ByVal imagen As Byte(), ByVal telefono As String, ByVal direccion As String, ByVal activo As Boolean, ByVal user_agr As String, ByVal fec_agr As Date, ByVal user_mod As String, ByVal fec_mod As Date, ByVal email As String, ByVal actualiza_costo_oc As Boolean, ByVal color As Integer, ByVal Codigo As String)
        Me.IdPropietario = IdPropietario
        Me.IdEmpresa = IdEmpresa
        Me.IdTipoActualizacionCosto = IdTipoActualizacionCosto
        Me.Contacto = contacto
        Me.Nombre_comercial = nombre_comercial
        Me.Imagen = imagen
        Me.Telefono = telefono
        Me.Direccion = direccion
        Me.Activo = activo
        Me.User_agr = user_agr
        Me.Fec_agr = fec_agr
        Me.User_mod = user_mod
        Me.Fec_mod = fec_mod
        Me.Email = email
        Me.Actualiza_costo_oc = actualiza_costo_oc
        Me.Color = color
        Me.Codigo = Codigo
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMSWebAPI/Entity/Propietario/clsBePropietarios.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Entity.Propietario
{
    public class clsBePropietarios : ICloneable
    {
        [Column("IdPropietario"), DisplayName("IdPropietario")]
        public int IdPropietario { get; set; } = 0;

        [Column("IdEmpresa"), DisplayName("IdEmpresa")]
        public int IdEmpresa { get; set; } = 0;

        [Column("IdTipoActualizacionCosto"), DisplayName("IdTipoActualizacionCosto")]
        public int IdTipoActualizacionCosto { get; set; } = 0;

        [Column("contacto"), DisplayName("Contacto")]
        public string Contacto { get; set; } = string.Empty;

        [Column("nombre_comercial"), DisplayName("Nombre Comercial")]
        public string Nombre_Comercial { get; set; } = string.Empty;

        [Column("imagen"), DisplayName("Imagen")]
        public byte[] Imagen { get; set; } = Array.Empty<byte>();

        [Column("telefono"), DisplayName("Teléfono")]
        public string Telefono { get; set; } = string.Empty;

        [Column("direccion"), DisplayName("Dirección")]
        public string Direccion { get; set; } = string.Empty;

        [Column("activo"), DisplayName("Activo")]
        public bool Activo { get; set; } = false;

        [Column("user_agr"), DisplayName("Usuario Agregado")]
        public string user_agr { get; set; } = string.Empty;

        [Column("fec_agr"), DisplayName("Fecha Agregado")]
        public DateTime fec_agr { get; set; } = DateTime.Now;

        [Column("user_mod"), DisplayName("Usuario Modificado")]
        public string user_mod { get; set; } = string.Empty;

        [Column("fec_mod"), DisplayName("Fecha Modificado")]
        public DateTime fec_mod { get; set; } = DateTime.Now;

        [Column("email"), DisplayName("Email")]
        public string email { get; set; } = string.Empty;

        [Column("actualiza_costo_oc"), DisplayName("Actualiza Costo OC")]
        public bool actualiza_costo_oc { get; set; } = false;

        [Column("color"), DisplayName("Color")]
        public int color { get; set; } = 0;

        [Column("codigo"), DisplayName("Código")]
        public string codigo { get; set; } = string.Empty;

        [Column("sistema"), DisplayName("Sistema")]
        public bool sistema { get; set; } = false;

        [Column("NIT"), DisplayName("NIT")]
        public string NIT { get; set; } = string.Empty;

        [Column("codigo_acceso"), DisplayName("Código Acceso")]
        public string codigo_acceso { get; set; } = string.Empty;

        [Column("clave_acceso"), DisplayName("Clave Acceso")]
        public string clave_acceso { get; set; } = string.Empty;

        [Column("IdBodegaAreaSAP"), DisplayName("Id Bodega Area SAP")]
        public int IdBodegaAreaSAP { get; set; } = 0;

        [Column("es_consolidador"), DisplayName("Es Consolidador")]
        public bool es_consolidador { get; set; } = false;

        public clsBePropietarios() { }

        public object Clone()
        {
            return MemberwiseClone();
        }
    }
}